docs/guides/views.rst

```diff
@@ -295,6 +295,34 @@ If your entity list will display the entity owners, you can improve performance
 
 See also :doc:`check this page out first </design/database>`.
 
+Since 1.11, you can define an alternative view to render list items using ```'item_view'``` parameter.
```

Comment:
Is a there a consensus on this? I don't want to this 'As of string' holding back the entire PR